CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributor Guidelines
+
+Thank you for your interest in contributing to the [Web Components Community Group](https://www.w3.org/community/webcomponents/)!
+
+Contributions to this repository are intended to become part of documents governed by the: 
+
+  * [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20200915/)
+  * [Software and Document License](https://www.w3.org/Consortium/Legal/copyright-software)
+
+To make substantive contributions to specifications, you must either participate
+in the relevant W3C Working Group or make a non-member patent licensing commitment.
+
+## Issue disclosure and discussion
+
+The first step for any substantive contribution is to either:
+
+  1. [Find an existing issue](https://github.com/w3c/csswg-drafts/issues) directly related to the contribution
+  2. [Add a new issue](https://github.com/w3c/csswg-drafts/issues/new)
+
+Issues are where individual reports and Community Group discussions come together such 
+that the eventual consensus can be turned into official language.
+
+## Normative and/or substantive contributions
+
+The Web Components Community Group operates via the consensus of its membership, and discusses all significant matters prior to implementation.
+
+<!-- @Westbrook would this be a good place to include the working group meeting links and cadence details?
```

Comment:
That sounds great. Something like:
```
The Community Group, aside from managing issues and PRs on GitHub, discusses recommendations, goals, and planning via events available on [this shared calendar](https://calendar.google.com/calendar/u/0/embed?src=o25bim5rvcu42mfnqilirpmp44@group.calendar.google.com).
```
I wish there was an easy way to leave a slack invite, but they all generally go stale, right?

Reply:
> Each invite link can be used by up to 2,000 people.

I think that will work long enough 😬
Once we 2000+ people then we probably can invest in automating this further

Reply:
Addressed this in c6dded6f1bbf7bfaf895f8acc1eb907bf5f42a8a

I wasn't able to ["edit link settings"](https://slack.com/intl/en-il/help/articles/201330256-Invite-new-members-to-your-workspace#share-an-invitation-link) to create a permanent invite link